hosts_example.yml

```diff
@@ -195,6 +230,26 @@ kafka_broker:
   #   kafka_broker:
   #     datadir:
   #       - /var/lib/kafka/my-data
+  #
+  ##  To configure LDAP for RBAC enablement, you will need to provide the appropiate properties to connect to your LDAP server
+  ##  using the ldap_config varible under the vars block.  The following is an example:
+  #
+  #   ldap_config: |
+  #     ldap.java.naming.factory.initial=com.sun.jndi.ldap.LdapCtxFactory
+  #     ldap.com.sun.jndi.ldap.read.timeout=3000
+  #     ldap.java.naming.provider.url=ldap://ldap1:389
+  #     ldap.java.naming.security.principal=uid=mds,OU=rbac,DC=example,DC=com
+  #     ldap.java.naming.security.credentials=password
+  #     ldap.java.naming.security.authentication=simple
+  #     ldap.user.search.base=OU=rbac,DC=example,DC=com
+  #     ldap.group.search.base=OU=rbac,DC=example,DC=com
+  #     ldap.user.name.attribute=uid
+  #     ldap.user.memberof.attribute.pattern=CN=(.*),OU=rbac,DC=example,DC=com
+  #     ldap.group.name.attribute=cn
+  #     ldap.group.member.attribute.pattern=CN=(.*),OU=rbac,DC=example,DC=com
+  #     confluent.security.event.logger.enabled=false
```

Comment:
this should be `confluent.security.event.logger.enable` (without `d` at the end) - and it shouldn't be part of ldap config, it should be enabled/disabled via a boolean var, as other settings are.

Reply:
We removed the confluent.security.event.logger.enable variable from our examples, you are right it does not need to be in the ldap config bit! 

You can toggle it the usual way with kafka_broker.properties dictionary